yarn-project/simulator/src/client/private_execution.test.ts

```diff
@@ -111,7 +111,7 @@ describe('Private Execution test suite', () => {
   const runSimulator = ({
     artifact,
     args = [],
-    msgSender = AztecAddress.ZERO,
+    msgSender = AztecAddress.fromField(new Fr(Fr.MODULUS - 1n)),
```

Comment:
NB: This change isn't actually required for the tests to pass (this test doesn't reach the kernels), it's just there to keep in line with `Fr.MAX` as the new default `msg_sender` value.